src/Js/utils/commentPop.js

```diff
@@ -0,0 +1,132 @@
+import getComments from './getComments.js';
+import postComment from './postComment.js';
+import commentsCounter from './commentsCounter.js';
+import updateComments from './updateComments.js';
+import createTableRow from './createTableRow.js';
+
+const commentPop = (meal) => {
+  const { idMeal, strMeal, strMealThumb } = meal;
+
+  const commentsObject = {
+    numberOfComment: 0,
+  };
+
+  const overlay = document.createElement('section');
+  overlay.className = 'overlay';
+  overlay.id = idMeal;
+
+  const popUp = document.createElement('div');
+  popUp.className = 'pop-up';
+
+  const closeButton = document.createElement('button');
+  closeButton.className = 'close';
+  closeButton.innerHTML = '&times;';
+  closeButton.addEventListener('click', () => {
+    //  document.querySelector('.overlay').classList.add('hide');
```

Comment:
- [ ] Let's make our codebase clean by removing commented code. There are also a few comments at the end of the file.
Let's also fix that. 💪